docs/package/pre-built.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: default
+title: "Pre-built unikernels"
+description: "Packaging pre-built unikernels"
+---
+
+# Packaging pre-built unikernels for `urunc`
+
+In this page we will explain the process of packaging an existing / pre-built
+unikernel as an OCI image with the necessary annotations for `urunc`. As an
+example, we will use a network example over
+[MirageOS](https://github.com/mirage) from
+[mirage-skeleton](https://github.com/mirage/mirage-skeleton/tree/main/device-usage/network)
+targeting [Solo5-hvt](https://github.com/Solo5/solo5).
+
+For simply packaging pre-built unikernel images, we can use both
+[bunny](https://github.com/nubificus/bunny) and
+[bunix](https://github.com/nubificus/bunix).
+
+> **NOTE**: The below steps can be easily adjusted to any pre-built unikernel image.
+
+## Using `bunny`
+
+In the case of [bunny](https://github.com/nubificus/bunny) and pre-built
+unikernel images, we can use both supported file syntaxes: a) `bunnyfile` and
+b) the Dockerfile-like syntax.
+
+### Using a `bunnyfile`
+
+In order to package an existing pre-built unikernel image with [bunny](https://github.com/nubificus/bunny) and a
+`bunnyfile` we can define the `bunnyfile` as:
+
+```
+#syntax=harbor.nbfc.io/nubificus/bunny:0.0.2
+version: v0.1
+
+version: v0.1
+
+platforms:
+  framework: mirage
+  monitor: hvt
+  architecture: x86
+
+kernel:
+  from: local
+  path: network.hvt
+
+cmdline: ""
+```
+
+In the above file we specify the followings:
+
+- We want to package a [MirageOS](https://github.com/mirage) unikernel that
+  will execute on top of [Solo5-hvt](https://github.com/Solo5/solo5) over x86
+  architecture.
+- We want to use the `network.hvt` binary as the unikernel to boot.
+- We do not specify any command line, since the unikernel does not necessary require one.
```

Comment:
I think the word "necessary" should be "necessarily" to correctly function as an adverb in this context.

Reply:
Yes, you are correct.